src/types.h

```diff
@@ -480,33 +481,30 @@ std::ostream &operator<<(std::ostream &os, ProbeType type);
 struct ProbeItem
 {
   std::string name;
-  std::string abbr;
+  std::unordered_set<std::string> aliases;
   ProbeType type;
+  bool show_in_kernel_list = false;
```

Comment:
Would be good to add short comment describing semantics

Reply:
Also, should we add `show_in_userspace_list` as well? So that we can get rid of `is_userspace_probe()` for similar reasons